javascript-timers/interval.js

```diff
@@ -0,0 +1,20 @@
+function beginCountDown() {
+  var h1 = document.querySelector('h1');
+  if (h1.textContent === '4') {
+    var three = h1.textContent = '3';
+    return three;
+  } else if (h1.textContent === '3') {
+    var two = h1.textContent = '2';
+    return two;
+  } else if (h1.textContent === '2') {
+    var one = h1.textContent = '1';
+    return one;
+  } else if (h1.textContent === '1') {
+    var earthBelow = h1.textContent = '~Earth Beeeellooww Us~';
+    return earthBelow;
+  }
+
+}
+
+setInterval(beginCountDown, 1000);
+clearInterval(2);
```

Comment:
- The interval's id will be different each time, don't hard-code it.
- The interval should only be cleared after the last text is displayed.